packages/pn-pa-webapp/src/pages/components/NewNotification/PreliminaryInformations.tsx

```diff
@@ -57,7 +54,7 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
     subject: yup.string().required(`${t('subject')} ${tc('required')}`),
     physicalCommunicationType: yup.string().required(),
     paymentMode: yup.string().required(),
-    group: groups.length > 0 && !isAdmin ? yup.string().required() : yup.string(),
```

Comment:
@AndreaCimini90 why this condition has been removed? groups management should have been fixed in branch main. Maybe some merging is necessary

Reply:
Yes! now it's ok